charts/airflow/templates/_helpers/validate-values.tpl

```diff
@@ -165,8 +165,8 @@
     {{- if .Values.postgresql.enabled }}
     {{ required "If `externalDatabase.host` is set, then `postgresql.enabled` should be `false`!" nil }}
     {{- end }}
-    {{- if not (has .Values.externalDatabase.type (list "mysql" "postgres")) }}
-    {{ required "The `externalDatabase.type` must be one of: [mysql, postgres]!" nil }}
+    {{- if not (has .Values.externalDatabase.type (list "mysql" "postgres" "mssql")) }}
```

Comment:
We should add another value-validation that prevents you using `externalDatabase.type = mssql` when `airflow.legacyCommands = true`. We can add the check to `_helpers/validate-values.tpl` after [line 10](https://github.com/airflow-helm/charts/blob/df99a33e3278de9f6935e747e8317794966ed979/charts/airflow/templates/_helpers/validate-values.tpl#L10).